ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -53,10 +128,29 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       $.ajaxSetup({async: true});
```

Comment:
This is removed, we shouldn't need it anymore. I added `TestUtils.waitForAjax` to allow for asynchronous testing of AJAX calls.